### DISCUSSION TITLE
Welcome to kunge-cybersecurity-learning Discussions!

### DISCUSSION THREAD

**Discussion by sechelper** [sechelper/kunge-cybersecurity-learning#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adityamishra1006** [comment]
Hello everyone, I am in my second year of college and want to explore and learn about the cybersecurity. Please help me in my journey of the cybersecurity. If at the beginning I need any courses or any books you guys can suggest then it will help me out. Thanks a lot.
